### PR TITLE
Add Onboarding Documentation

### PR DESCRIPTION
**Pull request by cpunt**
## What type of PR is this? (check all applicable)
- [X] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Description
Add onboarding documentation

## Manual test
- [X] 👍 yes
- [ ] 🙅 no

## Added tests?
- [ ] 👍 yes
- [X] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?
- [ ] 📜 README.md
- [X] 🙅 no documentation needed

### CODE REVIEW

File: docs/onboarding-tech/2-product.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+date: 2024-02-12
+category: document
+author:  Samson Rinaldi
+order: 2
+---
+
+# Our Products
+
+Since 2015, Jangala has been developing and implementing internet access solutions to support the delivery of critical social services to marginalised communities. We utilise a variety of self-built and off-the-shelf techniques and technologies to ensure we can provide the best possible service to our users both now and in the future. Our products cater to a diverse range of needs, from easy-to-manage and scalable networks that can serve hundreds or even thousands of users, to at-home safe Wi-Fi that enables providers to monitor and manage connections.
```
Comment (lePereT) [inline docs/onboarding-tech/2-product.md:10]: we're trying to not use the word marginalised, and instead using the phrase 'digitally excluded' :)

File: docs/onboarding-tech/2-product.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+Along with our product offerings, we provide a range of services, including site assessments, network design, hardware support, partner training, and cloud infrastructure. Currently, our core products—Big Box, Get Box, and the soon-to-be-released Lock Box—form the foundation of our connectivity and content infrastructure. As we move forward, we remain dedicated to expanding our offerings by introducing new products and developing our existing solutions to meet the ever-changing needs of our users.
```
Comment (lePereT) [inline docs/onboarding-tech/2-product.md:12]: For now, let's exclude Lock Box, until we're on firmer ground with it

File: docs/onboarding-tech/3-network.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+order: 3
+---
+
+# Networking: How We're Connected
+
+Welcome to a quick dive into how networking powers connectivity at Jangala! Whether you're tech-savvy or just curious about the technical underpinnings, this guide will provide you with key insights into how our products share connectivity.
+
+## Wi-Fi Connectivity
+
+Our devices broadcast Wi-Fi at two main frequencies: 2.4GHz and 5GHz, which are part of the radio band of the electromagnetic spectrum. The 2.4GHz band offers extensive coverage and greater range, making it ideal for reaching through walls and other barriers. In contrast, the 5GHz band provides faster data speeds, perfect for high-bandwidth activities like streaming and gaming.
```
Comment (lePereT) [inline docs/onboarding-tech/3-network.md:14]: It might be good to include pictures of the spectrum

Comment (lePereT) [reply docs/onboarding-tech/3-network.md:14]: Also the 5GHz band is wider and so can afford larger channels. For the same channel sizes the speeds at 2.4 and 5 are the same

File: docs/onboarding-tech/3-network.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+## Backhaul Options
+
+To connect to the broader internet, our devices use three types of backhaul: cable, satellite, and cellular towers. Each has its advantages — cable is reliable for steady connections, satellite covers wide areas beyond other infrastructures, and cellular offers flexibility with mobile access. This diversity ensures that our devices maintain connectivity in various settings.
```
Comment (lePereT) [inline docs/onboarding-tech/3-network.md:18]: Order them the opposite way around. Also it's not strictly true to say "3" for example, point to point radios aren't any of the 3. Maybe say 'primarily'

File: docs/onboarding-tech/3-network.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+## IP Addresses
+
+Each device connected to our Wi-Fi is assigned an IP address, akin to a postal address on the internet. This ensures that data packets are accurately delivered to and from your device, whether it's communicating within your home network or across the internet.
+
+## Routing and Data Retrieval
+
+When you send a request from your device, it is routed through the network to access the internet. Initially, the request moves to a Wide Area Network (WAN), reaches a DNS server to translates the domain name into an IP address, and then proceeds to the appropriate data center to retrieve the requested data. The response travels back along this path to reach your device, guided by the IP address.
+
+Through understanding these fundamental elements—from Wi-Fi transmission and backhaul options to IP addresses and routing—you'll appreciate how our devices facilitate reliable and efficient internet connections. This knowledge is essential as it ensures every interaction with our technology is smooth and productive. Welcome to the interconnected world of Jangala, where advanced networking brings your digital world to life.
```
Comment (lePereT) [inline docs/onboarding-tech/3-network.md:28]: This sounds very ChatGPT, put this in our own words

File: docs/onboarding-tech/2-product.md (hunk 2 above)
Comment (cpunt) [reply docs/onboarding-tech/2-product.md]: done

File: docs/onboarding-tech/3-network.md (hunk 3 above)
Comment (cpunt) [reply docs/onboarding-tech/3-network.md]: done

File: docs/onboarding-tech/3-network.md (hunk 4 above)
Comment (cpunt) [reply docs/onboarding-tech/3-network.md]: done

File: docs/onboarding-tech/3-network.md (hunk 5 above)
Comment (cpunt) [reply docs/onboarding-tech/3-network.md]: done

**Comment by cpunt** [comment]
Closing pr as moved documents to google sites

**Pull request closed without merging**